cdap-ui/app/directives/my-pipeline-scheduler/my-pipeline-scheduler.js

```diff
@@ -340,7 +341,8 @@ MyPipelineSchedulerCtrl.$inject = ['moment', 'myHelpers', 'CronConverter', '$sco
         onClose: '&',
         startSchedule: '&',
         isDisabled: '@',
-        suspendSchedule: '&'
+        suspendSchedule: '&',
+        validToSchedule: '@'
```

Comment:
It would be better to just pass boolean value rather than as a string, then you can just simply use the scope variable in the template without extra assignment in the controller